clusterplugin/config.go

```diff
@@ -66,6 +66,12 @@ type Cluster struct {
 
 	// LocalImagesPath path to local archive images to load into containerd from the filesystem  start
 	LocalImagesPath string `yaml:"local_images_path,omitempty" json:"local_images_path,omitempty"`
+
+	// ClusterRootPath path to a custom root directory
+	ClusterRootPath string `yaml:"clusterRootPath,omitempty" json:"clusterRootPath,omitempty"`
```

Comment:
I see this got introduced but I don't see it being used @kpiyush17

Reply:
we havent finished the changes to consume it. But it will be used in all providers e.g provider-kubeadm, provider-k3s

Reply:
removed this field as its not needed actually